substrate/frame/revive/src/lib.rs

```diff
@@ -548,6 +548,22 @@ pub mod pallet {
 	#[pallet::storage]
 	pub(crate) type OriginalAccount<T: Config> = StorageMap<_, Identity, H160, AccountId32>;
 
+	#[pallet::genesis_config]
+	#[derive(frame_support::DefaultNoBound)]
+	pub struct GenesisConfig<T: Config> {
+		/// Genesis mapped accounts
+		pub mapped_accounts: Vec<T::AccountId>,
+	}
+
+	#[pallet::genesis_build]
+	impl<T: Config> BuildGenesisConfig for GenesisConfig<T> {
+		fn build(&self) {
+			for id in &self.mapped_accounts {
+				T::AddressMapper::map(id).expect("Account not already mapped; qed");
```

Comment:
This is not really a proof. You don't have control over the genesis here. This is not test code. Let's not introduce panics. Why can't we just filter here instead of in the preset?